src/app/crawlstart/crawlstart.component.html

```diff
@@ -0,0 +1,309 @@
+<nav class="navbar navbar-default nav-about">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type = "button" class = "navbar-toggle"
+              data-toggle = "collapse" data-target = "#dropmenu">
+        <span class = "icon-bar"></span>
+        <span class = "icon-bar"></span>
+        <span class = "icon-bar"></span>
+      </button>
+      <a class="navbar-brand advsearch-navbar" href="//susper.com">
+        <img alt="brand" class="navbar-logo" src="../../assets/images/susper.svg">
+      </a>
+    </div>
+    <div class="collapse navbar-collapse" id="dropmenu">
+      <ul class="nav navbar-nav navbar-collapse navbar-right">
+        <li><a routerLink="/terms">Terms</a></li>
+        <li><a routerLink="/about">About</a></li>
+        <li><a routerLink="/contact">Contact</a></li>
+      </ul>
+    </div>
+  </div>
+</nav>
+
+
+
+<div class="container">
+  <h3 style="color: #dd4b39;font-family: Arial,SansSerif">Expert Crawl Start</h3>
+  <p>
+    You can define URLs as start points for Web page crawling and start crawling here.
+    "Crawling" means that YaCy will download the given website, extract all links in it and then download the content behind these links.
```

Comment:
Please change the message to:
"Crawling" implies YaCy will download the given website, extract all its links and then download the content behind these links.

This change is optional and only my suggestion, so you can ignore it if you find it unnecessary. :sweat_smile: